pkg/runtime/kubernetes/utils.go

```diff
@@ -0,0 +1,86 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubernetes
+
+import (
+	"fmt"
+
+	"net"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/sealerio/sealer/common"
+	"github.com/sealerio/sealer/utils/exec"
+	osi "github.com/sealerio/sealer/utils/os"
+	"github.com/sealerio/sealer/utils/ssh"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+// VersionCompare :if v1 >= v2 return true, else return false
+func VersionCompare(v1, v2 string) bool {
```

Comment:
i am ok with this func, but It would be great if we have a version struct to this.    
```shell
v1:=Version("v1")
v2:=Version("v2")
v1.Compare(v2)
```